pyispyb/dependencies.py

```diff
@@ -45,7 +45,10 @@ def permission(permission: str):
 
     async def with_permission() -> bool:
         if permission not in g.permissions:
-            raise HTTPException(status_code=403, detail="Not Authorised")
```

Comment:
I would leave this as returning "Not Authorised" but log an error / warning with your new message. This avoids leaking potentially sensitive information, but it can be dug out from logs